src/SignalR/clients/cpp/src/signalrclient/hub_connection_impl.cpp

```diff
@@ -383,8 +395,10 @@ namespace signalr
                         std::make_exception_ptr(
                             hub_exception(utility::conversions::to_utf8string(message.at(_XPLATSTR("error")).serialize()))));
                 }
-
-                set_result(json::value::null());
+                else
+                {
+                    set_result(json::value::null());
```

Comment:
Should this be `web::json::value()` instead? The [docs](https://microsoft.github.io/cpprestsdk/classweb_1_1json_1_1value.html) seem to indicate that `value()` is just a null value and `null()` is "A JSON null value".